### PR TITLE
Adding myAccount Password Docs

### PR DESCRIPTION
**Pull request by jakebacher2-okta**
## Description:
- **What's changed?** 
  - Adding in docs for myAccount API password endpoints
- **Is this PR related to a Monolith release?**  N/A

### Resolves:

* [OKTA-502543](https://oktainc.atlassian.net/browse/OKTA-502543)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md [hunk 1]
```diff
@@ -1253,6 +1257,243 @@ curl -XPUT 'https://${yourOktaDomain}/idp/myaccount/profile' -H 'Authorization:
 }
 ```
 
+### Get My Password
+
+<ApiOperation method="get" url="/idp/myaccount/password" />
+
+Retrieves the current user's password status.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md:1264]:
```suggestion
Retrieves the current user's password status
```

File: packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md [hunk 2]
```diff
@@ -1253,6 +1257,243 @@ curl -XPUT 'https://${yourOktaDomain}/idp/myaccount/profile' -H 'Authorization:
...
+
+> **Note:**  This only returns information about the password, not the password itself
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md:1266]:
```suggestion
> **Note:**  This operation only returns information about the password, not the password itself.
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md](https://github.com/okta/okta-developer-docs/blob/b499d923820fb61e61f08665faf89ef5f2f92b26/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fmyaccount%2Findex.md) | 77 | 161 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9b144624-a27d-47b4-830b-59e4bfada9d7) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.22b9e9e9-14ea-4174-9366-f98757c993d4?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md](https://github.com/okta/okta-developer-docs/blob/1ef51367b472af503a628c0205ca22c93ca17030/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fmyaccount%2Findex.md) | 77 | 160 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0fd01e3f-4409-47cb-94b4-4963b7d26a44) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.89d40da6-b580-4a33-b01b-3a4447691dfc?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

File: packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md [hunk 3]
```diff
@@ -1253,6 +1257,243 @@ curl -XPUT 'https://${yourOktaDomain}/idp/myaccount/profile' -H 'Authorization:
...
+
+This API requires a [My Password Request object](#my-password-request-object) as its request body.
+
+#### Response body
+
+The requested [My Password object](#my-password-object)
+
+#### Error Responses
+
+If the password supplied is invalid, the response returns a 400 BAD REQUEST with error code E0000001
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md:1348]:
```suggestion
If the password supplied is invalid, the response returns a 400 BAD REQUEST with error code E0000001.
```

File: packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md [hunk 4]
```diff
@@ -1253,6 +1257,243 @@ curl -XPUT 'https://${yourOktaDomain}/idp/myaccount/profile' -H 'Authorization:
...
+}
+```
+
+### Delete My Password
+
+Deletes the current user's enrolled password
+
+<ApiOperation method="delete" url="/idp/myaccount/password" />
+
+Deletes the password for the current user.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md:1460]:
```suggestion
```

File: packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md [hunk 5]
```diff
@@ -1253,6 +1257,243 @@ curl -XPUT 'https://${yourOktaDomain}/idp/myaccount/profile' -H 'Authorization:
...
+```bash
+curl -v -X DELETE \
+-H "Authorization: Bearer ${token}" \
+-H "Accept: application/json; okta-version=1.0.0" \
+"https://${yourOktaDomain}/idp/myaccount/password"
+```
+
+##### Response
+
+Returns an empty HTTP 204 status code response.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md:1489]:
```suggestion
Returns an empty HTTP 204 status code response
```

File: packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md [hunk 6]
```diff
@@ -1253,6 +1257,242 @@ curl -XPUT 'https://${yourOktaDomain}/idp/myaccount/profile' -H 'Authorization:
...
+    }
+}
+```
+
+### Delete My Password
+
+Deletes the current user's enrolled password
+
+<ApiOperation method="delete" url="/idp/myaccount/password" />
+
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md:1459]:
```suggestion
```

**Review by brentschaus-okta — APPROVED** [review]
LGTM

**Pull request merged**